src/index.html

```diff
@@ -6,7 +6,71 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page">
+    <div class="container">
+      <div class="calendar calendar__month--length--29">
```

Comment:
month-length and start-day are modifiers of calendar, they should:

- be placed in the same block - calendar
- be modifiers, not elements
- have correct naming style - month-length have 1 dash, because it's modifier name